styles/styles.css

```
@@ -1356,16 +1392,19 @@ body {
 .button-load {
   display: block;
   min-width: 620px;
-  line-height: 60px;
+  min-height: 60px;
+  list-style: none;
+  line-height: 26px;
   border-radius: 4px;
   border: none;
   text-transform: uppercase;
-  margin: 30px auto;
+  margin: 29px auto;
   text-align: center;
   font-size: 20px;
   font-weight: 700;
   color: #ffffff;
   background-color: #83b3d3;
+  font-family: "PT Sans", "Aria", sans-serif;
```

```suggestion
  font-family: inherit;
```
чтобы это не писать каждый раз, лучше сделать так где это нужно